templates/new_home/contact.html

```diff
@@ -0,0 +1,164 @@
+              </a>
+          </p>
+        </div>
+        </div>
+      </div>
+  
+    </div>
+  
+    <div class="col-md-6 px-0">
+      <div class="shadow-lg p-3 mb-5 bg-white card-rounder">
+        <div class="card-body mt-3">
+          <form action={% url 'feedbacks:contact' %} method="POST" id="contact-form" onsubmit="return validateForm()">
+  
+            <div id="error-display">  </div>
+            
+            <div class="form-group font-weight-bold">
+              <label class="ml-1" style="font-size: large;" for="full_name">Name</label>
+              <input type="text" class="form-control form-control-lg cstm-input" id="full_name" name="full_name" placeholder="Your Name...">
+            </div>
+            
+            <div class="form-group font-weight-bold">
+              <label class="ml-1" style="font-size: large;" for="phone">Phone</label>
+              <input type="text" class="form-control form-control-lg cstm-input" id="phone" name="phone" placeholder="Your Phone No...">
+            </div>
+            
+            <div class="form-group font-weight-bold">
+              <label class="ml-1" style="font-size: large;" for="email_id">Email</label>
+              <input type="email" class="form-control form-control-lg cstm-input" id="email_id" name="email_id" placeholder="Your Email...">
+            </div>
+        
+            <div class="form-group font-weight-bold">
+              <label class="ml-1" style="font-size: large;" for="msg">Message</label>
+              <textarea class="form-control form-control-lg cstm-input" name="msg" id="msg" placeholder="Write Something..." rows="5"></textarea>
+            </div>
+    
+            <div class="text-center">
+              <button type="submit" class="btn btn-cstm btn-get-started scrollto btn-lg px-4">Submit</button>
+            </div>
+    
+          </form>
+        </div>
+      </div>
+    </div>
+  
+  </div>
+</div>
+
+<script>
+    // Here is a js function to validate contact form here
+    function validateForm(){
+    let name = document.getElementById("full_name");
+    let phone = document.getElementById("phone");
+    let email_id = document.getElementById("email_id");
+    let msg = document.getElementById("msg");
+    let error_message = document.getElementById("error-display");
+
+    if(name.value=='' || name.value==null || name.value.length<3){
+      error_message.innerText = 'Please enter a valid Name...';
+      name.focus();
+      return false;
+    }
+
+    else if(!(isPhone(phone.value))){
+      error_message.innerText = 'Please enter a valid Phone Number...';
+      phone.focus();
+      return false;
+    }
+    
+    else if(!(isEmail(email_id.value))){
+      error_message.innerText = 'Please enter a valid Email id...';
+      email_id.focus();
+      return false;
+    }
+
+    else if(msg.value=='' || msg.value==null || msg.value.length<6){
+      error_message.innerText = 'Please enter a valid Message...';
```

Comment:
If you are throwing an error for `values.length` you should also inform them about it. Like the error message can be
`Please enter a valid message. (It should be more than 10 characters)`. (Same for number and name)

Also, you can remove `...` (three dots) from all the errors and change them to `.` (single dot).

Reply:
Done changes !